batch/batch/driver/scheduler.py

```diff
@@ -43,21 +44,28 @@ def __init__(self, app):
         self.inst_pool = app['inst_pool']
         self.async_worker_pool = AsyncWorkerPool(parallelism=100, queue_size=100)
         self.exceeded_shares_counter = ExceededSharesCounter()
+        self.task_manager = aiotools.BackgroundTaskManager()
```

Comment:
Same question about whether this should come from the app?

Reply:
In this case, the scheduler needs a shutdown method anyway (it should have been shutting down the pool), so it seems best to keep all the scheduler's shutdown logic in one place.